### DISCUSSION TITLE
error message when start up with plugin configurations

### DISCUSSION THREAD

**Discussion by Jax-Tsai-zero** [hyprwm/Hyprland#7542] [Q&A]
I have my plugins enabled with exec dispatcher and configured them with keyword "plugin" in my .hypr config file. However, every time I start up the error message comes up first, and only after a second for loading plugins, everything goes on track, which is really annoying. Did I do something wrong? Much appreciated if anyone could help :pray:

**Comment by fufexan** [comment]
Why not load plugins with `plugin = /path/to/libplugin.so` directly?
